make/utils.cmake

```diff
@@ -221,4 +221,40 @@ if (UNIX)
     append_property_string(TARGET ${target} LINK_FLAGS "${ldflags}")
   endfunction (set_preferred_base_start_and_end)
 
+  function (check_avx_processor_and_compiler_support out)
```

Comment:
I tried to unresolve the prior comment but it won't let me reply.  I still am not a fan of checking in code that is not executed at all when it's easy to fix that.  For all we know it won't work on Travis at all b/c of some cmake version difference or missing CheckCSourceRuns or who knows what.  Please add a call to it: you will want to store the result in a variable anyway.  Otherwise it does not seem worth committing separately if we just have to change it all due to some CI failure.

Reply:
Added UNIX test calls to top level for BUILD_TESTS.